### PR TITLE
Functions to parse layout.json

### PR DESCRIPTION
**Pull request by GewoonJaap**
- loadJSON(path);
- getJSONData();
- doesPlaneInfoExist(jsonObject);
- addPlaneInfo(jsonObject);
- deletePlaneInfo(texturePath);

### CODE REVIEW

File: src/helpers/MSFS/layoutParser.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  let data = getJSONData();
+  for (let i = 0; i < data.content.length; i++) {
+    if (data.content[i].path == texturePath) {
+      data.content.splice(i, 1);
+      break;
+    }
+  }
+
+  fs.writeFileSync(FILE_PATH, JSON.stringify(data, null, 4));
+}
```
Comment (NinjaLabs-Dev) [inline src/helpers/MSFS/layoutParser.js:86]: Make sure to add a new line at the end 😄

**Review by NinjaLabs-Dev — CHANGES_REQUESTED** [review]

File: src/helpers/MSFS/layoutParser.js (hunk 1 above)
Comment (NinjaLabs-Dev) [reply src/helpers/MSFS/layoutParser.js:86]: Apart from that looks great gg

Comment (GewoonJaap) [reply src/helpers/MSFS/layoutParser.js:86]: Reee, will do :D

Comment (GewoonJaap) [reply src/helpers/MSFS/layoutParser.js:86]: There is already a new line after line 86

**Review by davwheat — APPROVED** [review]

**Pull request merged**